README.md

```diff
@@ -587,6 +587,94 @@ Once you opt-in to use additional OCI hooks, any new orchestrated containers wil
+* `settings.oci-defaults.capabilities.fsetid`: process capability `CAP_FSETID`
+* `settings.oci-defaults.capabilities.ipc-lock`: process capability `CAP_IPC_LOCK`
+* `settings.oci-defaults.capabilities.ipc-owner`: process capability `CAP_IPC_OWNER`
+* `settings.oci-defaults.capabilities.kill`: process capability `CAP_KILL`
+* `settings.oci-defaults.capabilities.lease`: process capability `CAP_LEASE`
+* `settings.oci-defaults.capabilities.linux-immutable`: process capability `CAP_LINUX_IMMUTABLE`
+* `settings.oci-defaults.capabilities.mac-admin`: process capability `CAP_MAC_ADMIN`
+* `settings.oci-defaults.capabilities.mac-override`: process capability `CAP_MAC_OVERRIDE`
+* `settings.oci-defaults.capabilities.mknod`: process capability `CAP_MKNOD`
+* `settings.oci-defaults.capabilities.net-admin`: process capability `CAP_NET_ADMIN`
+* `settings.oci-defaults.capabilities.net-bind-service`: process capability `CAP_NET_BIND_SERVICE`
+* `settings.oci-defaults.capabilities.net-broadcast`: process capability `CAP_NET_BROADCAST`
+* `settings.oci-defaults.capabilities.net-raw`: process capability `CAP_NET_RAW`
+* `settings.oci-defaults.capabilities.perfmon`: process capability `CAP_PERFMON`
+* `settings.oci-defaults.capabilities.setgid`: process capability `CAP_SETGID`
+* `settings.oci-defaults.capabilities.setfcap`: process capability `CAP_SETFCAP`
+* `settings.oci-defaults.capabilities.setpcap`: process capability `CAP_SETPCAP`
+* `settings.oci-defaults.capabilities.setuid`: process capability `CAP_SETUID`
+* `settings.oci-defaults.capabilities.sys-admin`: process capability `CAP_SYS_ADMIN`
+* `settings.oci-defaults.capabilities.sys-boot`: process capability `CAP_SYS_BOOT`
+* `settings.oci-defaults.capabilities.sys-chroot`: process capability `CAP_SYS_CHROOT`
+* `settings.oci-defaults.capabilities.sys-module`: process capability `CAP_SYS_MODULE`
+* `settings.oci-defaults.capabilities.sys-nice`: process capability `CAP_SYS_NICE`
+* `settings.oci-defaults.capabilities.sys-pacct`: process capability `CAP_SYS_PACCT`
+* `settings.oci-defaults.capabilities.sys-ptrace`: process capability `CAP_SYS_PTRACE`
+* `settings.oci-defaults.capabilities.sys-rawio`: process capability `CAP_SYS_RAWIO`
+* `settings.oci-defaults.capabilities.sys-resource`: process capability `CAP_SYS_RESOURCE`
+* `settings.oci-defaults.capabilities.sys-time`: process capability `CAP_SYS_TIME`
+* `settings.oci-defaults.capabilities.sys-tty-config`: process capability `CAP_SYS_TTY_CONFIG`
+* `settings.oci-defaults.capabilities.syslog`: process capability `CAP_SYSLOG`
+* `settings.oci-defaults.capabilities.wake-alarm`: process capability `CAP_WAKE_ALARM`
+
+##### OCI Defaults: Resource Limits
+
+Each of the `resource-limits` settings below contain two numeric fields: `.hard-limit` and `.soft-limit`, which are **32-bit unsigned integers**.
+
+Please see the `getrlimit` linux manpage for meanings of `.hard-limit` and `.soft-limit`.
```

Comment:
Maybe link to https://man7.org/linux/man-pages/man7/capabilities.7.html for the man page? This link is already referenced in code in the settings model crate